tsconfig.json

```diff
@@ -36,9 +36,9 @@
     // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
     "moduleDetection": "force", /* Control what method is used to detect module-format JS files. */
     /* Modules */
-    "module": "commonjs", /* Specify what module code is generated. */
+    "module": "ES2022", /* Specify what module code is generated. */
     "rootDir": "./src/", /* Specify the root folder within your source files. */
-    // "moduleResolution": "node10",                     /* Specify how TypeScript looks up a file from a given module specifier. */
+    "moduleResolution": "node",                          /* Specify how TypeScript looks up a file from a given module specifier. */
```

Comment:
Hoowee this looks like a potentially far-reaching change that's probably outside of the scope of this PR - an accidental commit?

Reply:
Reverted.